### ISSUE TITLE
[labs/motion] @animate directive: wrong animationFrame call 

### ISSUE THREAD

**Issue by d--j** [lit/lit#4370]
### Which package(s) are affected?

Motion (@lit-labs/motion)

### Description

`animationFrame` is a function that returns a Promise:

```ts
export const animationFrame = () =>
  new Promise((resolve) => requestAnimationFrame(resolve));
```

But here 

https://github.com/lit/lit/blob/b2030b44e75d32b2e8433a8977848a672c995014/packages/labs/motion/src/animate.ts#L328

it looks like it was forgotten to call the function.

Since the function is not *thenable* and not a Promise this line just waits for a microtask instead of an animation frame as far as I know.

Since it works the way it is (waiting for a microtask instead of an animation frame) I suggest changeing that line to something more explicit:
```ts
await Promise.resolve()
```

---

The second usage of `animationFrame` does call the function:

https://github.com/lit/lit/blob/b2030b44e75d32b2e8433a8977848a672c995014/packages/labs/motion/src/animate.ts#L405

### Reproduction

N/A

### Workaround

N/A

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

1.0.6 & main

### Browser/OS/Node environment

N/A

**Comment by augustjk** [comment]
Good catch. @sorvell do you know what the intended behavior is? It seems like the original intention was to await on an animation frame, but it's also been like that for a long time, so is just a microtask sufficient?
